packages/fetchai/connections/p2p_libp2p/connection.py

```diff
@@ -173,7 +173,7 @@ def make_acn_envelope_message(envelope: Envelope) -> bytes:
         buf = acn_msg.SerializeToString()
         return buf
 
-    def make_agent_record(self) -> AcnMessage.AgentRecord:  # type: ignore
+    def make_agent_record(self) -> AcnMessage.AgentRecord:  # type: ignore  # pragma: nocover
```

Comment:
We should have a test for this. Why does this not get used in integration tests? That's worrying.

Reply:
looks like this code is not used anywhere, so removed